lib/protoflow-blocks/src/system.rs

```diff
@@ -242,14 +247,19 @@ impl TextBlocks for System {
     }
 
     fn concat_strings_by(&mut self, delimiter: &str) -> ConcatStrings {
-        self.0.block(ConcatStrings::with_system(self, Some(delimiter.to_string())))
+        self.0.block(ConcatStrings::with_system(
+            self,
+            Some(delimiter.to_string()),
+        ))
     }
 
     fn split_string(&mut self, delimiter: &str) -> SplitString {
-        self.0.block(SplitString::with_system(self, Some(delimiter.to_string())))
+        self.0
+            .block(SplitString::with_system(self, Some(delimiter.to_string())))
     }
 
     fn split_string_whitespace(&mut self) -> SplitString {
-        self.0.block(SplitString::with_system(self, Some(r"\s+".to_string())))
+        self.0
+            .block(SplitString::with_system(self, Some(r"\s+".to_string())))
```

Comment:
@evren-okcu @AlexSanches1 Here we see the problem of not having run `cargo fmt` on both your PRs: this pending merge conflict (with regards to #17) didn't need to happen. Kindly do make sure to `cargo fmt` going forward!

Two options here:
- @evren-okcu You could restore these nonrelevant lines to their original state on your end (meaning they don't show up as changed in this PR), preventing the conflict.
- @AlexSanches1 If that's not done, you will need to handle the conflict with #19 on your end after this gets merged.

Reply:
@artob I have removed the merge commit which caused these problems, after @AlexSanches1's PR is merged, I get solve the merge commit.